lib/author.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/book'
+
+class Author
+  attr_reader :books
+  def initialize(author_hash)
+    @first_name = author_hash[:first_name]
+    @last_name = author_hash[:last_name]
+    @books = []
+  end
+
+  def name
+    @first_name + " " + @last_name
+  end
```

Comment:
good job calculating this here versus the book.author suggestion I made earlier, since we DO need to keep the first/last name below